make/modules/java.desktop/lib/ClientLibraries.gmk

```diff
@@ -23,16 +23,17 @@
 # questions.
 #
 
-include CopyFiles.gmk
+/*
+ * ===========================================================================
+ * (c) Copyright IBM Corp. 2024, 2024 All Rights Reserved
+ * ===========================================================================
+ */
```

Comment:
Comments begin with `#` in makefiles; we want this immediately after line 24:
```
# ===========================================================================
# (c) Copyright IBM Corp. 2024, 2024 All Rights Reserved
# ===========================================================================
```

Reply:
Fixed the comments.